qdyn/output.f90

```diff
@@ -581,6 +608,10 @@ subroutine ot_write(pb)
   character(len=100) :: tmp
   character(len=100), allocatable :: iot_name
 
+  ! CRP: to extract ivmax_fault and vmax_fault
+  ! integer, dimension(pb%nfault) :: ivmax_fault
+  ! double precision, dimension(pb%nfault) :: vmax_fault
```

Comment:
Why are these lines commented out?

Reply:
I think before on a first trial, I tried to initialize those variables inside the output module, while the ot series is being written, but at the end I do it inside of problem_class. I removed the lines.